playwright.config.ts

```diff
@@ -0,0 +1,107 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
```

Comment:
can you remove it. we have a config in the `backend` folder where we actually run the playwright config